pygeo/parameterization/DVGeo.py

```diff
@@ -600,6 +613,9 @@ def addPointSet(self, points, ptName, origConfig=True, **kwargs):
 
         """
 
+        # compNames is only needed for DVGeometryMulti, so remove it if passed
+        kwargs.pop("compNames", None)
```

Comment:
Just a minor concern. DVGeoMulti has more arguments than just `compNames`, like `applyIC` and `comm`. While no constraints that are calling `addPointSet` are including these parameters at the moment, they may. Should we pop them here? If not, the following call to `attachPoints` will fail. To avoid that, you are basically back to my Suggestion 2. Not sure if this is a concern. @sseraj you may have more insights into if that is possibly needed for the constraints when using DVGeoMulti.

Reply:
I am fine with popping all the unnecessary `kwargs` here

Reply:
I don't anticipate adding the other kwargs like `comm` and `applyIC` to the constraints. If we do end up adding them in the future, we can also pop them. I would like to keep this list to a minimum though just so we catch any potential errors.